pkg/image/validation/kubernetes.go

```diff
@@ -117,6 +113,48 @@ func validateNodes(k8s *image.Kubernetes) []FailedValidation {
 	return failures
 }
 
+func validateNetwork(k8s *image.Kubernetes) []FailedValidation {
+	var failures []FailedValidation
+	ip := k8s.Network.APIVIP
+
+	numNodes := len(k8s.Nodes)
+	if numNodes <= 1 {
+		// Single node cluster, node configurations are not required
+		return failures
+	}
+
+	if ip == "" {
+		failures = append(failures, FailedValidation{
+			UserMessage: "The 'apiVIP' field is required in the 'network' section when defining entries under 'nodes'.",
+		})
+	}
+
+	if ip != "" {
+		parsedIP, err := netip.ParseAddr(ip)
+		if err != nil {
+			failures = append(failures, FailedValidation{
+				UserMessage: fmt.Sprintf("Invalid APIVIP address %q for field 'apiVIP'.", ip),
+				Error:       err,
+			})
+		}
+
+		if !parsedIP.Is4() && !parsedIP.Is6() {
+			failures = append(failures, FailedValidation{
+				UserMessage: fmt.Sprintf("%q is not an IPV4 or IPV6 address, only IPV4 and IPV6 addresses are valid for field 'apiVIP'.", ip),
```

Comment:
Can we reduce the repetition?
```suggestion
				UserMessage: "Only IPv4 and IPv6 addresses are valid values for field 'apiVIP'.",
```